linux_os/guide/services/obsolete/r_services/no_rsh_trust_files/rule.yml

```diff
@@ -45,6 +45,8 @@ references:
     nist: CM-7(a),CM-7(b),CM-6(a)
     nist-csf: PR.AC-3,PR.IP-1,PR.PT-3,PR.PT-4
 
+machine: package[rsh-server]
```

Comment:
I think you want `machine` to be `platform` here.

Reply:
:facepalm: 

Fix pushed.

Thank you!